etc/docker/Dockerfile

```diff
@@ -28,23 +28,31 @@
 FROM ubuntu
 
 ENV DEBIAN_FRONTEND noninteractive
-ENV RELEASE 1
-ENV NAT64 1
-ENV DNS64 1
 
-COPY . /app
+RUN apt-get update && apt-get install -y \
+        git \
+        lsb-core \
+        sudo && \
+        apt-get autoremove -y -o APT::Autoremove::RecommendsImportant=0 -o APT::Autoremove::SuggestsImportant=0 && rm -rf /var/lib/apt/lists/*
 
+COPY . /app
 WORKDIR /app
+RUN git reset --hard && git clean -xffd
 
-RUN apt-get update && apt-get install -y git lsb-core sudo
+ENV RELEASE 1
+ENV NAT64 1
+ENV DNS64 1
 
-RUN git reset --hard && git clean -xffd
-RUN ./script/bootstrap && ./script/setup
-RUN chmod 644 /etc/bind/named.conf.options
-RUN mv ./script /tmp && mv ./etc /tmp
-RUN find . -delete && rm -rf /usr/include
-RUN mv /tmp/script . && mv /tmp/etc .
-RUN apt-get autoremove -y -o APT::Autoremove::RecommendsImportant=0 -o APT::Autoremove::SuggestsImportant=0 && rm -rf /var/lib/apt/lists/*
+RUN ./script/bootstrap && \
+    ./script/setup && \
+    chmod 644 /etc/bind/named.conf.options && \
```

Comment:
move this above COPY?

Reply:
could not be moved above COPY, as it is a file after bind9 is installed during script/bootstrap